content/millennium/r4/financial/general/insurance-plan.md

```diff
@@ -36,9 +36,10 @@ Search for InsurancePlans that meet supplied query parameters:
 
 ### Parameters
 
-Name                  | Required? | Type          | Description
-----------------------|-----------|---------------|-------------------------------------------------------
-`owned-by`            | Yes       | [`reference`] | Organization associated with the InsurancePlan.
+Name                  | Required?          | Type          | Description
+----------------------|--------------------|---------------|-------------------------------------------------------
+`_id`                 | This or `owned-by` | [`token`]     | The logical resource id associated with the resource.
```

Comment:
It doesn't look like 'token' is a link in the screen shot. Just want to make sure it is actually a link.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/817/commits/15c2ec2a0a95ece00eb19e4be3f6b42a093cb923#diff-505b119a76d7cbd4b9ad19f605bcfd1d674bfb0fca50632ebc7948b1bc50b3caR98 

Updated the screenshot as well.